language/gpt-oss/requirements.txt

```diff
@@ -0,0 +1,11 @@
+audioread>=2.1.9
```

Comment:
Please use == (we had very bad experience with using >= where the impl is broken after 1 round)
(you can use pip freeze in your env and copy the versions here)